include/ginkgo/core/log/logger.hpp

```diff
@@ -58,6 +58,11 @@ class PolymorphicObject;
 class Operation;
 class stopping_status;
 
+/**
+ * @brief The Stopping criterion namespace .
```

Comment:
Previously, you never had a space before the dot when doing the `@brief`:
```suggestion
 * @brief The Stopping criterion namespace.
```